monai/data/dataset.py

```diff
@@ -773,6 +778,12 @@ def __init__(
                 the dataset has duplicated items or augmented dataset.
             hash_func: if `hash_as_key`, a callable to compute hash from data items to be cached.
                 defaults to `monai.data.utils.pickle_hashing`.
+            runtime_cache: whether to compute cache at the runtime, default to `False` to prepare
+                the cache content at initializaiton, if `True`, it will cache during the first epoch
+                of model training, so it can start the first mini-batch earlier.
+                to execute runtime cache on GPU memory, must co-work with `monai.data.DataLoader`.
+            cache_rank: if in distributed data parallel, only compute the cache on `cache_rank` and
```

Comment:
I don't think we need to expose cache_rank as a user option. We can always use rank 0. Regardless of who allocates it, the shared cpu cache in cpu RAM, shared memory.  I do not see a use case when a user needs to change it

Reply:
Sure, removed this arg in the latest commit.

Thanks.